easycv/models/detection/detectors/yolox/yolox.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) 2014-2021 Megvii Inc And Alibaba PAI-Teams. All rights reserved.
```

Comment:
by detection/detectors/detection.py build yolox pipeline

Reply:
done

Reply:
yolox  refactor need to be done in next sprint